newswires/client/src/App.tsx

```diff
@@ -187,87 +189,83 @@ export function App() {
 				>
 					{!isPoppedOut && (
 						<EuiHeader position="fixed">
-							<EuiHeaderSection>
+							<EuiHeaderSection side={'left'}>
 								<EuiHeaderSectionItem>
 									<SideNav />
 								</EuiHeaderSectionItem>
+								<EuiShowFor sizes={['s', 'm', 'l', 'xl']}>
```

Comment:
this leaves the `xs` page layout without a `h1` landmark heading, unless the menu is opened.

as we've discussed, this seems reasonable overall given the space constraints, and likely expectations of users with narrow screens, but it feels like we should probably make sure there's always a `h1` on the page even if it's visually hidden, so perhaps we could add something like:

```
<ShowFor sizes={['xs'])>
    {!sideNavIsOpen && <ScreenReaderOnly>
        <h1>Newswires (under construction)</h1>
    </>}
</>
```